src/cc/frontends/clang/loader.cc

```diff
@@ -170,7 +180,7 @@ int ClangLoader::parse(unique_ptr<llvm::Module> *mod, TableStorage &ts,
   }
 
   // If all attempts to obtain kheaders fail, check for kheaders.tar.xz in sysfs
-  if (!is_dir(kpath)) {
+  if (!is_file(kpath + "/include/linux/kconfig.h")) {
```

Comment:
could you add a comment here on why checking `{kpath}/include/linux/kconfig.h`? Otherwise, people looking at code will get really puzzled.

Reply:
Sorry, got AFK, and then forgot a bit. Is newly added comment good enough?

Reply:
Sounds good to me.